TypeTreeDiffGUI/DropArea.xaml.cs

```diff
@@ -55,8 +57,26 @@ private void OnDropped(object sender, DragEventArgs e)
 		{
 			if (e.Data.GetDataPresent(DataFormats.FileDrop))
 			{
-				string[] files = (string[])e.Data.GetData(DataFormats.FileDrop);
-				EventFileDropped?.Invoke(files[0]);
+				string[] filePaths = ((string[])e.Data.GetData(DataFormats.FileDrop));
+
+				if (filePaths == null || filePaths.Length == 0)
+				{
+					throw new ArgumentException("No files or folders were dropped.");
+				}
+				if (filePaths.Length > 1)
+				{
+					throw new ArgumentException("Multiple files or folders were dropped but only one is supported.");
```

Comment:
I don't think ArgumentException is appropriate here. Just Exception should be fine. Other than that, this PR should be good.